### PR TITLE
feat: Add spec "parted__l" and enhance its parser

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

* Collect spec `parted__l`, and enhance its parser `PartedL` according to the command's output.
* Add spec for `sos_commands/block/parted_-s_*` files in sosreport, and add its parser `PartedSos`.

### CODE REVIEW

**Comment by JoySnow** [comment]
Hi @xiangce , @chenlizhong , Could you help review this MR?

File: insights/parsers/parted.py [hunk 1]
```diff
@@ -231,3 +168,189 @@ def parse_content(self, content):
...
+            except ParseException:
+                # Mute any exception from PartedDevice parsing.
+                # Keep only fully parsed Device into the final devices_info.
+                pass
+
+        self.devices_info = devices_info
+
+
+@parser(Specs.parted_s_sos)
+class PartedSos(CommandParser):
```
Comment (xiangce) [inline insights/parsers/parted.py:294]: I didn't find there are any GSS rules that used the original "Parted" parser , so it's not so necessary to add this one for SOS here on the core side. I mean, we can remove this one, and later if there are such requests from GSS side, we can also add it to the GSS models.

**Review by xiangce — CHANGES_REQUESTED** [review]

**Review by xiangce — APPROVED** [review]

File: insights/parsers/parted.py (hunk 1 above)
Comment (JoySnow) [reply insights/parsers/parted.py]: Removed the `PartedSos`.

File: insights/parsers/parted.py [hunk 2]
```diff
@@ -136,24 +75,22 @@ def get(self, item):
...
+class PartedDevice(object):
+    """Class to contain information for one device of ``parted`` command output.
 
-    The columns may vary depending upon the type of device.
+    The columns of partation table may vary depending upon the type of device.
 
     Attributes:
-        data (dict): Dictionary of information returned by ``parted`` command.
-        partitions (list): The partitions found in the output, as Partition
+        data (dict): Dictionary of information of this device section.
```
[the remark this answers is not in the document]
Comment (JoySnow) [reply insights/parsers/parted.py]: Updated as suggested.

**Review by chenlizhong — APPROVED** [review]

**Pull request merged**